pkg/events/events.go

```diff
@@ -0,0 +1,83 @@
+package events
```

Comment:
You might want to consider using the go-sdk for cloudevents:
https://github.com/cloudevents/sdk-go

Reply:
Thx, in fact I did and looking at `go.mod` for the SDK it pulls in a lot of dependencies [1] I did not want to bloat the final binary with. We still use the SDK in `events_test.go` to validate we conform to the v1 spec though. 

[1]
https://github.com/cloudevents/sdk-go/blob/master/go.mod